Source/EventHorizon/Consumer/Processing/EventsFromEventHorizonFetcher.cs

```diff
@@ -47,6 +47,10 @@ public async Task<Try<IEnumerable<StreamEvent>>> Fetch(StreamPosition streamPosi
         }
     }
 
+    /// <inheritdoc />
+    public async Task<Try<StreamPosition>> GetNextStreamPosition(CancellationToken cancellationToken)
+        => new NotImplementedException();
```

Comment:
I don't know if there is a "reason" kind of option here? But maybe it is useful for ourselves to remember why it is not implemented?

Reply:
```suggestion
        => new NotImplementedException("GetNextStreamPosition should never be used on this specific fetcher");
```